Makefile.am

```diff
@@ -116,9 +116,22 @@ test:
 	bin/urweb -boot -noEmacs -dbms sqlite -db $(TESTDB) -demo /Demo demo
 	rm -f $(TESTDB)
 	sqlite3 $(TESTDB) < demo/demo.sql
-	demo/demo.exe & echo $$! > $(TESTPID)
+	demo/demo.exe -a 127.0.0.1 & echo $$! > $(TESTPID)
```

Comment:
Should there also be a test that not passing any port works (and for CI, works even without IPv6 sort)?